bundles/action/org.openhab.action.pushover/src/main/java/org/openhab/action/pushover/internal/Pushover.java

```diff
@@ -233,105 +323,119 @@ public static boolean pushover(@ParamDoc(name = "apiKey", text = "Your applicati
             @ParamDoc(name = "urlTitle", text = "A title for your supplementary URL, otherwise just the URL is shown.") String urlTitle,
             @ParamDoc(name = "priority", text = "Send as -1 to always send as a quiet notification, 1 to display as high-priority and bypass the user's quiet hours, or 2 to also require confirmation from the user.") int priority,
             @ParamDoc(name = "sound", text = "The name of one of the sounds supported by device clients to override the user's default sound choice.") String sound) {
+        return pushover0(apiKey, user, message, device, title, url, urlTitle, priority, sound, null, null) != null;
```

Comment:
Can the builder pattern be applied in this situation, too? If so we can handle the complete preparation process in the `sendPushoverMessage()` method and don't need to define a `pushover0()` method with the complete set of different parameters. Wdyt?

Reply:
sure one could avoid the private method, you will save 2-3 lines of code.
the idea here is to have one private method, which does the whole hard implementing work, like 'pushover0', this method can be changed/refactored or what ever without any problems, since its private and we do not break the api
all other public methods are parts of the contract/api, they just use the private method with a minimum of own logic
thats simply, how i like to implement things
all a matter of taste though

Reply:
@cweitkamp are you ok with merging now?